docs/source/models/intervention_models/alzheimers/hypothetical_treatment.rst

```diff
@@ -30,4 +30,234 @@
+        neg [label="BBBM test negative [3 y]"]
+
+        wait [label="In treatment/ Waiting for treatment [6 mo]"]
+        no_treat [label="No treatment effect [permanent]"]
+
+        treat [label="Full treatment effect LONG [5 y]"]
+        wane [label="Waning treatment effect LONG (update each step) [9 y]"]
+
+        treat_short [label = "Full treatment effect SHORT [6 mo]"]
+        wane_short [label = "Waning treatment effect SHORT (update each step) [2.5 y]"]
+
+
+        el -> test [label = "tested"]
+        el -> el [label = "not tested"]
+
+        test -> pos  [label = "(90%)", style=dashed]
+        test -> neg [label = "(10%)", style=dashed]
+        neg -> el [label = "test re-eligible"]
+
+        pos -> wait [label = "decides to initiate treatment (I)", style=dashed]
+        pos -> no_treat [label = "decides not to initiate treatment (1 - I)", style=dashed]
+
+        wait -> treat [label = "completes (90%)"]
+        wait -> treat_short [label = "discontinues (10%)"]
+
+        treat_short -> wane_short
+        wane_short -> no_treat
+
+        treat -> wane 
+
+        wane -> no_treat
+    }
+
+The diagram above illustrates how a simulant should progress through the various testing and treatment related 
+states defined by the client. Each simulant may transition to a new state on each time step. 
+
+Most states have a fixed duration (a multiple of the 
+time step length) where simulants will transition after :math:`\text{duration} / \text{time step}` time steps. 
+The duration is marked in the state node in brackets eg [6 mo]. As desribed in the :ref:`testing intervention <alzheimers_testing_intervention_bbbm>`, 
+some simulants in the `BBBM test eligible` state may transition to tested immediately (low propensity value), some may always self-transition
+(ie, never get tested, high propensity value), and some may self-transition for some number of time steps but eventually transition to tested
+as a result of the time-specific testing rate increasing.
+
+Some states have zero duration, illustrated with a dashed box (rather than the solid ovals for states with nonzero durations). 
+Transitions from a state with zero duration are illustrated with a dashed line. If a simulant transitions to a zero-duration state 
+on a time step, they should also immediately continue to the next state during that same time step, as a part of the same transition.
+For example, a simulant in `BBBM test eligible` who is tested and moves to `BBBM test received` would then immediately move to one of 
+that state's two sinks, and would even move directly to another state during the same transition/ time step on a positive test.
+
+Below are tables with details on how to model these states and transitions, and necessary data values. 
+The value of :math:`i_{MCI}` in the :ref:`cause model <2021_cause_alzheimers_presymptomatic_mci_transition_data_table>` is now updated
+to be equal to :math:`h_{adj} = h_{MCI} * R_h`.
```

Comment:
```suggestion
to be equal to :math:`h_{adj} = h_{MCI} \cdot R_h`.
```
If we're going to use $\LaTeX$, we might as well use a standard multiplication symbol: either $\cdot$ (`\cdot`) or $\times$ (`\times`), or juxtaposition with no symbol in between.